group_members.go

```diff
@@ -0,0 +1,94 @@
+package sarama
+
+type GroupMemberMetadata struct {
```

Comment:
As for naming, we should probably call this `ConsumerGroupMemberMetadata`. This is a specialization of the generic group type.